openshift-ci/run_metallb_e2e.sh

```
@@ -24,7 +27,7 @@ elif [ "${IP_STACK}" = "v6" ]; then
 elif [ "${IP_STACK}" = "v4v6" ]; then
 	SKIP="$SKIP|IPV6"
 	export PROVISIONING_HOST_EXTERNAL_IPV4=${PROVISIONING_HOST_EXTERNAL_IP}
-	export PROVISIONING_HOST_EXTERNAL_IPV6=1111:1:1::1
+	export PROVISIONING_HOST_EXTERNAL_IPV6=$(nthhost "$EXTERNAL_SUBNET_V6" 1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)